### ISSUE TITLE
For cycle1 (april 2019)

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#273]
## Summary

> _Specify the total amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ requested: `3100`

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule (as still suggested on the slack #compensation channel)
- concerns april 2019, ie the 01/04/2019-30/04/2019 period.

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-488246551 | (BSQ requested = `2500`)
Evaluation is based and intended in line with : previous compensation requests and other support compensation requests.
Much extra support required this month mainly alas due to the scam. Triggers warnings about scalability imo.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = `200`)
Concerned monitored data files : https://github.com/HarryMacfinned/bisq-monitor
TL;DR: beginning of april was hectic for bitcoin. Bisq got some crazy fees from our provider, which in turn triggered Bisq's fees to the maximum (1000sat/byte). Ended in adjustment for the pricenodes.

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-488576063 | (BSQ requested = `200`)
TL;DR: only 2 brief incidents : one pricenode had timeout issues for some hours. Another pricenode was slightly incoherent for some hours.

NB1 : The monitoring role includes also manual sorting of false alarms, ie monitoring several Bisq specific pages + following of Tor metrics page + maillist
NB2 : All the alerts are given on the slack #bisq-monitor and #bisq-pricenode channels

## Contributions delivered
- Completed test script "National Currency Accounts" [`100 BSQ`]
- Completed test script "Altcoin Accounts" [`50 BSQ`]
- Testing (exploratory style) [`50 BSQ`]
-- [Appli not shuting down](https://github.com/bisq-network/bisq/issues/2634)
-- [Appli not shuting down](https://github.com/bisq-network/bisq/issues/2673)
-- [UI suggestion on showing security deposit](https://github.com/bisq-network/bisq/issues/2761)

I reported in the #testing slack channel and reports for scripts are available on https://bisq.ontestpad.com/
It's first time I ask compensation for testing. I didn't find a previous exact matching case to compare with (sorry if I missed it). I'm fully open to change the requested amounts in one direction or the other (but imo we must also not discourage other testers).

- Clearnet Bisq webpages monitoring : only some back-office alerts, reported in the #bisq-monitor and #bisq-markets channels [`0 BSQ`]
- https://github.com/bisq-network/bisq/issues/2625 (led to corrective PR) [`0 BSQ`]
- https://github.com/bisq-network/bisq/pull/2680 [`0 BSQ`]

## Contributions in progress
- I have another test script ongoing (trading), but alas not finished atm.
- I would like to jump from testing to directly proposing some PR ... but lacked time for that.
